src/components/atoms/BooleanCell/BooleanCell.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type BooleanCellProps = {
+    booleanVal?: boolean;
+    onOffVals: {
+        on: string,
+        off: string
+    }
```

Comment:
onOff seems a little out of place here when everything else is named around a boolean.

Reply:
As I mentioned below, I think I was influenced by a previous pattern in this case. We could definitely change the BooleanCell for clarity.